CHANGELOG.md

```diff
@@ -11,15 +11,28 @@ Note that Sockeye has checks in place to not translate with an old model that wa
 Each version section may have have subsections for: _Added_, _Changed_, _Removed_, _Deprecated_, and _Fixed_.
 
 ## [2.0.0]
+
 ### Changed
```

Comment:
I think it would be nice to use the PR descriptions on Distributed Training, Training Options, Docker build, Misc here as well with a nested list. We will need some 'release notes' document anyway.

Reply:
Expanding the changelog is a good idea.  We've kept it minimal to avoid clutter, but anyone actually reading it is probably interested in the details.